src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml

```diff
@@ -50,13 +50,12 @@ tests:
               minValue: 0
               maxValue: 32768
 
-    #Issue 18041
     - label: "TH reads from the DUT the (0xFFFB) AttributeList attribute"
       PICS: PICS_SKIP_SAMPLE_APP
       command: "readAttribute"
       attribute: "AttributeList"
       response:
-          value: [0, 7, 10, 13, 23, 26, 65528, 65529, 65531, 65532, 65533]
```

Comment:
what was 26 ?

Reply:
@jmeg-sfy its an optional attribute (26) SafetyStatus (O) we are just checking for Mandatory attributes for now.
![image](https://user-images.githubusercontent.com/102359958/174624491-910653eb-5956-4de8-930c-8b55bb88c3d9.png)